eclass/kernel-install.eclass

```diff
@@ -263,12 +263,14 @@ kernel-install_can_update_symlink() {
 	# strip KV_LOCALVERSION, we want to update the old kernels not using
 	# KV_LOCALVERSION suffix and the new kernels using it
 	symlink_ver=${symlink_ver%${KV_LOCALVERSION}}
+	# strip -p* revision
+	local symlink_ver_no_rev=${symlink_ver%-p[0-9]*}
```

Comment:
Admittedly, the stripping here may be a bit overzealous, but I don't really want to go down the `extglob` rabbit hole.

Reply:
Can we use `ver_cut` here?